### PR TITLE
global: fix the last modified conversion

### PR DESCRIPTION
**Pull request by krzysztof**
* Fixes the incorrect conversion of last modified timestamps
  from localtime to UTC.

Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -75,6 +75,7 @@
     extras_require['all'].extend(reqs)
 
 install_requires = [
+    'arrow>=0.8.0',
```
Comment (jirikuncar) [inline setup.py]: Are we sure that we want to introduce new dependency?

Comment (krzysztof) [inline setup.py]: alternative is to ``import calendar`` and use https://docs.python.org/2/library/calendar.html#calendar.timegm, but arrow seemed to be more abundant across other packages, so I'd vote yes for the consistency.

Comment (jirikuncar) [inline setup.py]: Can we go with `calendar` if it works for you?

**Comment by jirikuncar** [comment]
@krzysztof can you also add test case which would fail without your fix? Thanks

File: setup.py (hunk 1 above)
Comment (krzysztof) [inline setup.py]: :+1:

File: invenio_files_rest/storage/base.py [hunk 2]
```diff
@@ -28,8 +28,10 @@
 
 import hashlib
 import time
+from calendar import timegm
 from functools import partial
 
+
```
Comment (jirikuncar) [inline invenio_files_rest/storage/base.py]: ?

**Comment by jirikuncar** [comment]
LGTM

File: invenio_files_rest/storage/base.py (hunk 2 above)
Comment (krzysztof) [inline invenio_files_rest/storage/base.py]: Fixed the isort and the unit test, take a look.

**Comment by jirikuncar** [comment]
LGTM

**Pull request merged**